fixtures/unstable-async/time-slicing/README.md

```diff
@@ -18,7 +18,7 @@ There are also known bugs and inefficiencies in master so **don't use this fixtu
 # 1: Build react from source
 cd /path/to/react
 yarn
-yarn build dom,core,interaction,simple-cache-provider --type=NODE
```

Comment:
The same change could benefit the suspense demo readme too. And is there an interaction-package or was that an earlier name of schedule?

Reply:
Updated suspense readme. Not 100% sure if interaction was renamed.

Reply:
interaction is now part of schedule